blogs_notebooks/titanic_data_science_solutions_blog_1.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>Explain what we're going to do in the next cell with the data and what to focus on.  I looked at the print outs, but didn't realize that all we would be looking for was the print, as there seems to be no actual comparison below despite what was said above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>